BabelfishDump.spec

```diff
@@ -151,6 +151,15 @@ LD_LIBRARY_PATH=%{_builddir}/%{name}/src/interfaces/libpq $RPM_BUILD_ROOT/usr/bi
 %{_bindir}/bbf_dumpall
 
 %changelog
+* Fri Oct 17 2025 Yashneet Vinayak <yashneet@amazon.com> - 17.7-1
+- Unrestrict psql meta-commands for using backslash command for pg_dump and pg_dumpall in Babelfish
```

Comment:
This description is a bit confusing, lets write it as: `Unrestrict Babelfish specific psql meta-commands for bbf_dump and bbf_dumpall`